### PR TITLE
py-pymorph: new package

### PR DESCRIPTION
**Pull request by neilflood**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-pymorph/package.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+class PyPymorph(PythonPackage):
+    """This image morphology toolbox implements the basic binary and greyscale
+       morphology operations, working with numpy arrays representing images"""
+
+    homepage = "http://www.example.com"
+    url = "https://pypi.org/packages/source/p/pymorph/pymorph-0.96.tar.gz"
+
+    version('0.96', sha256='5dd648e4cb4c3495ee6031bc8020ed8216f3d6cb8c0dcd0427b215b75d7d29ad')
+
+    depends_on('py-setuptools', type='build')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-pymorph/package.py]: I would also add:
```python
depends_on('python@:2', type=('build', 'run'))
```
as this library is definitely not ready for Python 3. From the homepage, it actually sounds like this package is deprecated.

File: var/spack/repos/builtin/packages/py-pymorph/package.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPymorph(PythonPackage):
+    """This image morphology toolbox implements the basic binary and greyscale
+       morphology operations, working with numpy arrays representing images"""
+
+    homepage = "http://www.example.com"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-pymorph/package.py]:
```suggestion
    homepage = "http://luispedro.org/software/pymorph/"
```

File: var/spack/repos/builtin/packages/py-pymorph/package.py (hunk 1 above)
Comment (neilflood) [reply var/spack/repos/builtin/packages/py-pymorph/package.py]: Dammit, well spotted. I had already been wondering whether I should just tell the person using it to re-write with other packages like scipy.ndimage. I will do this instead, and close this pull request. Better not to clutter Spack with packages we already know are obsolete - clutter is likely to become a problem into the future. 

Many thanks for noticing that.

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-pymorph/package.py]: No problem. If they have some aversion to `scipy.ndimage` you might want to take a look at http://luispedro.org/software/mahotas/. It seems to be the same or similar library but with a C++ implementation behind the scenes making it much faster.

**Pull request closed without merging**